tests/test_databases.py

```diff
@@ -202,6 +208,16 @@ def run_sync(*args, **kwargs):
             assert result["text"] == "example2"
             assert result["completed"] == False
 
+            # fetch_val()
+            query = "SELECT completed FROM notes WHERE text = :text"
+            result = await database.fetch_val(query=query, values={"text": "example1"})
+            assert result == True
+            query = "SELECT completed FROM notes WHERE text = :text"
+            result = await database.fetch_val(
+                query=query, values={"text": "example1"}, column="completed"
+            )
```

Comment:
Should this read `query = "SELECT * FROM ..."` and then use `column="completed"`?
It's not obvious at first sight why adding `column=...` makes sense here otherwise?

Reply:
You're right, I changed it